fetch.bs

```diff
@@ -5153,12 +5276,14 @@ these steps:
      <a>filtered response</a>; otherwise to <var>response</var>'s
      <a for="filtered response">internal response</a>.
 
+     <li><p>If <var>response</var>'s <a for=response>type</a> is "<code>error</code>", then
+     return the result of running <a>HTTPS upgrade fallback</a> given <var>request</var>
+     and a <a>network error</a>.
```

Comment:
This doesn't seem like the right place to be doing this. This means that whenever we get an error back from a service worker we'd do this?

Reply:
Removed the service worker related changes altogether due to the conclusion in https://docs.google.com/document/d/1J5AkicHtSsCq2YVzLYwcFNj27Ah0-9VVTvkyevYxxPQ/edit?usp=sharing

(The conclusion being SWs can't intercept HTTP navigations, and this is a navigation-only feature, therefore we don't need to do any SW specific handling)